src/instance.ts

```diff
@@ -364,7 +364,7 @@ export function readConfigFile(
 	}
 }
 
-let EXTENSIONS = /\.tsx?$|\.jsx?$/
+let EXTENSIONS = /\.tsx?$|\.jsx?|.json$/
```

Comment:
The change needs to escape the period to work correctly.

Reply:
corrected, thanks for review!